scripts-available/CDB_GhostTables.sql

```diff
@@ -0,0 +1,117 @@
+-- Table to store the transaction id from DDL events to avoid multiple executions
+CREATE TABLE IF NOT EXISTS cartodb.cdb_ddl_execution(txid integer PRIMARY KEY, tag text);
+
+-- Enqueues a job to run Ghost tables linking process for the provided user_id
+CREATE OR REPLACE FUNCTION _CDB_LinkGhostTables(username text, db_name text, ddl_tag text) 
+RETURNS void
+AS $$
+  if not username:
+    return
+
+  if 'json' not in GD:
+    import json
+    GD['json'] = json
+  else:
+    json = GD['json']    
+
+  error = ''
```

Comment:
Why are you declaring this variable here empty?

Reply:
Copied from another place. Removing it as it's not needed...